### DISCUSSION TITLE
Cannot whitelist folder in /home/data

### DISCUSSION THREAD

**Discussion by eloyesp** [netblue30/firejail#5959] [Q&A] [answered]
I store most data (audio and video for example) on a different disk that is out of my home folder, and I create symlinks to access the files with ease.

The problem I'm having is that I'm not able to whitelist anything out my home folder, symlinks do not work.

So I want a custom profile that allows full access to `/home/me/music/thisapp` that is actually stored on `/home/data/music/thisapp` (the external disk). Is there a way to accomplish that?

I've tested with:

```
whitelist /home/me/music/thisapp # => this creates a fake folder on the sandbox and is deleted on quit
whitelist /home/data/music/thisapp # => this do nothing
```

**Comment by ghost** [comment, marked as answer]
> whitelist /home/data/music/thisapp # => this do nothing

By default, `only` the current user's home directory is visible. The `allusers` option makes  `all` directories under /home visible inside the sandbox. So it looks like you need that.

**Reply by kmk3** [reply]
This is correct, though it might be better to just do the mounting in /mnt/data
instead of /home/data so that `allusers` is not necessary.
